exe/kubernetes-deploy

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
+require 'kubernetes-deploy'
+
+options = {
+  namespace: ARGV[0],
+  context: ARGV[1],
+  environment: ENV['ENVIRONMENT'],
+  current_sha: ENV['REVISION'],
+  kubeconfig_path: ENV["KUBECONFIG"],
+}
+
+options[:template_dir] = ENV['K8S_TEMPLATE_FOLDER'] || File.expand_path("./config/deploy/#{options.fetch(:environment)}")
```

Comment:
The `K8S` in the name here is somewhat redundant, and another environment variable uses `KUBE` as a prefix. I'd say just remove it.

Where does the default come from? As far as I understand, it's not used in core in this manner.